ksql-engine/src/test/resources/schema-validation-tests/avro-basic.json

```diff
@@ -0,0 +1,17 @@
+{
+  "tests": [
+    {
+      "name": "basic avro schema",
```

Comment:
shouldn't one of the inputs be the data format, ie. avro, json, or delimited? It would be nice to receive source data in any format and any schema, and also test writing in different formats with different schemas. Having a framework for this will help when we start adding more schemas.